src/net.h

```diff
@@ -506,6 +506,8 @@ class CNodeStats
     double dMinPing;
     std::string addrLocal;
     CAddress addr;
+    // Bind address that this connection came in on (for incoming connections)
```

Comment:
Yes, addrLocal is pretty much useless in most cases, as it's what is sent by the peer. Wish we could rename that one. But yes it needs to be documented.